api/app/src/main/kotlin/packit/controllers/RunnerController.kt

```diff
@@ -57,4 +58,10 @@ class RunnerController(private val runnerService: RunnerService)
     {
         return ResponseEntity.ok(runnerService.submitRun(submitRunInfo))
     }
+
+    @GetMapping("/status/{taskId}")
+    fun getJobStatus(@PathVariable taskId: String): ResponseEntity<RunInfoDto>
```

Comment:
maybe `getTaskStatus` is a better name for consistency everywhere else

Reply:
done, thanks